pkgs/development/nim-packages/naylib/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+  , nimPackages
+  , fetchFromGitHub
+  , raylib
+  , libX11
+  , libXi
+  , libXxf86vm
+  , libxkbcommon
+  , libXcursor
+  , libXrandr 
+  , libXinerama 
+}:
+
+nimPackages.buildNimPackage rec {
+  pname = "naylib";
+  version = "4.6.2";
+
+  src = fetchFromGitHub {
+    owner = "planetis-m";
+    repo = "${pname}";
```

```suggestion
    repo = "naylib";
```
semantically different